test/integration/cat/test_cat.py

```diff
@@ -6,51 +6,65 @@
 
 from utils import run_command_and_get_output
 
+TEST_CAT_ARGS_CCONFIG = ("args, cat_result, found, not_found")
 
-def test_cat_unset_arg(tt_cmd, tmp_path):
-    # Testing with unset .xlog or .snap file.
-    cmd = [tt_cmd, "cat"]
-    rc, output = run_command_and_get_output(cmd, cwd=tmp_path)
-    assert rc == 1
-    assert re.search(r"it is required to specify at least one .xlog or .snap file", output)
-
-
-def test_cat_non_existent_file(tt_cmd, tmp_path):
-    # Testing with non-existent .xlog or .snap file.
-    cmd = [tt_cmd, "cat", "path-to-non-existent-file"]
-    rc, output = run_command_and_get_output(cmd, cwd=tmp_path)
-    assert rc == 1
-    assert re.search(r"No such file or directory", output)
-
-
-def test_cat_snap_file(tt_cmd, tmp_path):
-    # Copy the .snap file to the "run" directory.
-    test_app_path = os.path.join(os.path.dirname(__file__), "test_file", "test.snap")
-    shutil.copy(test_app_path, tmp_path)
 
-    # Testing .snap file.
-    cmd = [
-        tt_cmd, "cat", "test.snap", "--show-system",
-        "--space=320", "--space=296", "--from=423", "--to=513"
-        ]
-    rc, output = run_command_and_get_output(cmd, cwd=tmp_path)
-    assert rc == 0
-    assert re.search(r"lsn: 423", output)
-    assert re.search(r"lsn: 512", output)
-    assert re.search(r"space_id: 320", output)
-    assert re.search(r"space_id: 296", output)
+def make_test_cat_args_param(
```

Comment:
Question: what is the benefit of this extra layer of abstraction?

Reply:
We set default values for all parameters, and have a possibility skip them in some test cases.

Reply:
Are there cases where a parameter can be omitted and default values used instead?
Even so, it is more transparent to have an **explicit** indication that the input is _“empty”_ data than to do so implicitly.

Reply:
> Are there cases where a parameter can be omitted and default values used instead?

Yes. I removed this code from places, where it does not needed.